modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValuesValidatorImpl.java

```diff
@@ -225,22 +225,23 @@ protected boolean isNull(
 
 		Value value = ddmFormFieldValue.getValue();
 
-		if (value == null) {
+		if (value == null || value.getAvailableLocales().isEmpty()) {
 			return true;
 		}
 
 		DDMFormFieldValueAccessor<?> ddmFormFieldValueAccessor =
 			getDDMFormFieldValueAccessor(ddmFormField.getType());
 
-		for (Locale availableLocale : value.getAvailableLocales()) {
-			if (ddmFormFieldValueAccessor.isEmpty(
-					ddmFormFieldValue, availableLocale)) {
+			for (Locale availableLocale : value.getAvailableLocales()) {
+				if (!ddmFormFieldValueAccessor.isEmpty(
+						ddmFormFieldValue, availableLocale)) {
 
-				return true;
+					return false;
+				}
 			}
-		}
 
-		return false;
+			return true;
+
```

Comment:
@MarcellGyopos, why did you decide to change the current logic? Check for `value.getAvailableLocales().isEmpty()` on the previous `if` isn't enough?

Reply:
Oh... After some re-reading, I believe that I got your point. 😁 I'll forward your pull to the DDM/DDL official reviewer, once he already back from PTO. 🙂

Reply:
Thank you @natocesarrego :)